cudax/test/CMakeLists.txt

```diff
@@ -93,4 +93,9 @@ foreach(cn_target IN LISTS cudax_TARGETS)
   cudax_add_catch2_test(test_target memory_resource ${cn_target}
     memory_resource/any_resource.cu
   )
+
+  cudax_add_catch2_test(test_target async_memory_resource ${cn_target}
+    memory_resource/async_memory_pool.cu
+    memory_resource/async_memory_resource.cu
+  )
```

Comment:
mild preference for making these tests part of the existing `memory_resource` target.

Reply:
Oh yeah, those were just added recently